Consul.Test/OperatorTest.cs

```diff
@@ -330,5 +330,35 @@ public async Task Operetor_AreaDelete()
 
             Assert.Null(req.Response);
         }
+
+        [Fact]
+        public async Task OperatorUsage_GetState()
+        {
+            var result = await _client.Operator.OperatorUsageGetState(QueryOptions.Default);
+
+            Assert.NotNull(result);
+            Assert.NotNull(result.Response);
+            Assert.NotNull(result.Response.Usage);
+            Assert.True(result.Response.Usage.Count > 0, "Expected at least one datacenter in usage response");
+
+            // Verify we have at least one datacenter with valid data
+            var firstDatacenter = result.Response.Usage.First();
+            Assert.False(string.IsNullOrEmpty(firstDatacenter.Key), "Datacenter name should not be empty");
+            Assert.NotNull(firstDatacenter.Value);
+
+            var usage = firstDatacenter.Value;
+
+            // Verify ConnectServiceInstances exists and has valid data
+            Assert.NotNull(usage.ConnectServiceInstances);
+            Assert.True(usage.ConnectServiceInstances.ConnectNative >= 0);
+            Assert.True(usage.ConnectServiceInstances.ConnectProxy >= 0);
+            Assert.True(usage.ConnectServiceInstances.IngressGateway >= 0);
+            Assert.True(usage.ConnectServiceInstances.MeshGateway >= 0);
+            Assert.True(usage.ConnectServiceInstances.TerminatingGateway >= 0);
```

Comment:
These checks do not add any value (they would be always true, even if these field didn't exist in the response), I guess they should be removed.